addon/mixins/unsubscribe-route.js

```diff
@@ -1,20 +0,0 @@
-import { deprecate } from "@ember/application/deprecations";
-import Mixin from "@ember/object/mixin";
-
-export default Mixin.create({
-  resetController() {
-    deprecate(`The \`UnsubscribeRoute\` mixin is deprecated, use \`RouteQueryManager\` instead.`, false, {
-      id: 'ember-apollo-client.deprecate-unsubscribe-route',
-      until: '1.0.0',
-    });
-
-    this._super(...arguments);
-
-    // If the model came from an apollo query, it will have an
-    // _apolloUnsubscribe function to stop it from receiving store updates.
-    let unsubscribe = this.get('controller.model._apolloUnsubscribe');
```

Comment:
@bgentry we must remove `subscription.unsubscribe()` from mixins as `ObjectQueryManager` right?

```js
_apolloUnsubscribe() {
    subscription.unsubscribe();
 },
```